pkgs/tools/typesetting/tex/tetex/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchurl, flex, bison, zlib, libpng, ncurses, ed, automake }:
+{ lib, stdenv, fetchurl, flex, bison, zlib, libpng12, libpng ? libpng12, ncurses, ed, automake }:
```

Comment:
```
error: builder for '/nix/store/hapr60a6qhjwd2hhf5ljz2334pmfp5ll-tetex-3.0.drv' failed with exit code 2;
       last 10 log lines:
       > writepng.c:152:45: error: invalid use of incomplete typedef 'png_info' {aka 'struct png_info_def'}
       >   152 |             pdfbuf[pdfptr++] = png_info(img)->palette[i].blue;
       >       |                                             ^~
```

The issue is explained here https://github.com/NixOS/nixpkgs/issues/259879#issuecomment-1827743352